### PR TITLE
Simplify default serializer deprecation docs

### PR DESCRIPTION
**Pull request by jenweber**
I found this section a little bit difficult to understand. I rearranged the explanation and simplified some of the language.

### CODE REVIEW

File: content/ember-data/v3/default-serializer.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ since: '3.12.0'
 
 ### deprecate adapter.defaultSerializer fallback
```
Comment (jaredgalanis) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
### Deprecate adapter.defaultSerializer fallback
```

File: content/ember-data/v3/default-serializer.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ since: '3.12.0'
 ---
 ## Default serializer deprecations
 
-### deprecate adapter.defaultSerializer fallback
+### Deprecate adapter.defaultSerializer fallback
 
-Previously, if no application or type-specific serializer was specified, the store would attempt to lookup a serializer via the `defaultSerializer` property on the type's adapter. This behavior is deprecated in favor of explicitly defining a type-specific serializer or application serializer.
+Moving forward, every app or addon that uses Ember Data must have a serializer explicitly defined.
+
+Previously, if no application or model-type-specific serializer was specified, the store would attempt to lookup a serializer via the `defaultSerializer` property on the type's adapter. This behavior is deprecated in favor of explicitly defining a type-specific serializer or application serializer.
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
Previously, if no application or model-type-specific serializer was specified, the store would attempt to look up a serializer via the `defaultSerializer` property on the type's adapter. This behavior is deprecated in favor of explicitly defining a model-type-specific serializer or application serializer.
```

File: content/ember-data/v3/default-serializer.md [hunk 3]
```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
...
+If all the adapters in your app are the same kind (such as JSONAPI, REST, or JSON), you should create an
+application serializer to match.
 
-Previously, when no type-specific serializer, application serializer, or adapter defaultSerializer had been defined by the app, the `-default` serializer would be used which defaulted to the `JSONSerializer`. This behavior is deprecated in favor of explicitly defining an application or type-specific serializer as described below.
+For example, if you use only JSONAPI adapters, creating the file below in `app/serializers/application.js`
+will resolve the deprecation:
 
-## clearing these deprecations
+```javascript
+// app/serializers/application.js
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
```javascript {data-filename=app/serializers/application.js
```

File: content/ember-data/v3/default-serializer.md [hunk 4]
```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
...
 
-More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+export { default } from '@ember-data/serializer/json-api';
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]: Were these module paths available in 3.12?

File: content/ember-data/v3/default-serializer.md [hunk 5]
```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
...
+```
 
-If a specific model type requires custom serialization, a type-specific serializer can be created. A single `application` serializer can be used for any model types not requiring custom serialization. To define a type-specific serializer, create an `app/serializers/[type].js` with the following:
+If your app uses different adapter types for different models, you should make one serializer for each model type. For example, if a certain model uses the `RESTAdapter`, create an `app/serializers/[model-type].js` file with the following:
 
 ```js
     import RESTSerializer from '@ember-data/serializer/rest';
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]: This is pre-existent, but I don't think it should have indentation.

File: content/ember-data/v3/default-serializer.md [hunk 6]
```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
...
-
-More information on adding these serializers can be found in "clearing these deprecations" below.
+### Clearing these deprecations
 
-### -default serializer fallback in store.serializerFor
+If all the adapters in your app are the same kind (such as JSONAPI, REST, or JSON), you should create an
+application serializer to match.
 
-Previously, when no type-specific serializer, application serializer, or adapter defaultSerializer had been defined by the app, the `-default` serializer would be used which defaulted to the `JSONSerializer`. This behavior is deprecated in favor of explicitly defining an application or type-specific serializer as described below.
+For example, if you use only JSONAPI adapters, creating the file below in `app/serializers/application.js`
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
For example, if you use only JSONAPI adapters, creating the following application serializer
```

File: content/ember-data/v3/default-serializer.md [hunk 7]
```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
...
-## clearing these deprecations
+```javascript
+// app/serializers/application.js
 
-More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+export { default } from '@ember-data/serializer/json-api';
+```
 
-If a specific model type requires custom serialization, a type-specific serializer can be created. A single `application` serializer can be used for any model types not requiring custom serialization. To define a type-specific serializer, create an `app/serializers/[type].js` with the following:
+If your app uses different adapter types for different models, you should make one serializer for each model type. For example, if a certain model uses the `RESTAdapter`, create an `app/serializers/[model-type].js` file with the following:
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]: What if you give a concrete model name as an example and use the `data-filename` facility to show the path?

File: content/ember-data/v3/default-serializer.md [hunk 8]
```diff
@@ -41,8 +43,4 @@ If a specific model type requires custom serialization, a type-specific serializ
     }
 ```
 
-Defining a serializer for the entire application can be done by adding the file `app/serializers/application.js` with the following:
-
-```js
-    export { default } from '@ember-data/serializer/json-api';
-```
+More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
```
Comment (locks) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [official guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
```

Comment (locks) [reply content/ember-data/v3/default-serializer.md]: English question, does anyone have a good mnemonic for "on the guides" vs "in the guides"?

File: content/ember-data/v3/default-serializer.md (hunk 4 above)
Comment (jaredgalanis) [reply content/ember-data/v3/default-serializer.md]: @locks i think so 🤷‍♀ 
https://github.com/emberjs/data/blob/e0732481885abde4eb187d3a1573276374068a2f/packages/serializer/addon/json-api.js#L412

Comment (locks) [reply content/ember-data/v3/default-serializer.md]: https://guides.emberjs.com/v3.14.0/models/defining-models/
We seem to only have switched the defaults in 3.15 though. Should we align these with the default at the time of the deprecation?

File: content/ember-data/v3/default-serializer.md (hunk 8 above)
Comment (jaredgalanis) [reply content/ember-data/v3/default-serializer.md]: I think I prefer in, but don't have super strong opinions. Probably would make more sense to use on if were saying `on the Guides website`, but either way I think it def would sound better if we unified in/on b/w the api docs and guides here.

File: content/ember-data/v3/default-serializer.md (hunk 4 above)
Comment (jaredgalanis) [reply content/ember-data/v3/default-serializer.md:32]: yeah makes sense

**Pull request merged**